packages/reports/src/Containers/__tests__/open-positions.spec.tsx

```diff
@@ -0,0 +1,475 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router-dom';
+import { isDesktop, isMobile } from '@deriv/shared';
+import { TPortfolioPosition } from '@deriv/stores/types';
+import { mockStore } from '@deriv/stores';
+import OpenPositions from '../open-positions';
+import ReportsProviders from '../../reports-providers';
+
+const data_list = 'DataList';
+const future_time = Math.floor(Date.now() / 1000) + 5000;
+const options_position = {
+    contract_info: {
```

Comment:
Will it make sence to use `mockContractInfo` here?

Reply:
Same in other places

Reply:
it only makes sense with in `open-positions.spec.tsx` and in `open-positions-table.spec.tsx`, i replaced it there, thank you ) However, a real `contract_info` object is required in `mobile-row-renderer.spec.tsx`